client/my-sites/exporter/advanced-settings.jsx

```diff
@@ -91,3 +95,13 @@ export default React.createClass( {
 		);
 	}
 } );
+
+const mapStateToProps = ( state ) => {
+	const siteId = state.ui.selectedSiteId;
+	return {
+		siteId: siteId,
+		shouldShowPlaceholders: !advancedSettings( state, siteId )
```

Comment:
Very minor thing: Here and in one or two spots later, we're missing a space after the `!`.